demos-europe/demosplan-core
# feat(refs T12070):  New back to top button

**Ticket:** https://yaits.demos-deutschland.de/T12070



This replaces the old Back to top Button.

On every Page in the admin-area this button should appear. But only after scrolling 100vh.
The Position "allways" fit. Even after resizing and on small screens and with a large menu and at the bottom of the page.


### How to review/test


Goto a admin-page with enogh content (like a-table with some statements.) Scroll 100vh and see if the btt-Button does what its suppose to do.


### Linked PRs (optional)

:warning: 
Only works after https://github.com/demos-europe/demosplan-ui/pull/364 is Published.
--> https://github.com/demos-europe/demosplan-core/pull/1712



### Tasks (optional)


### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
